RemoteWellnessSupportApp/OnBoarding/OnboardingScreenView.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  OnboardingScreenView.swift
+//  RemoteWellnessSupportApp
+//
+//  Created by 岩本雄貴 on 2024/01/31.
+//
+
+import SwiftUI
+
+struct OnboardingScreenView: View {
+    @AppStorage("hasCompletedOnboarding") var hasCompletedOnboarding: Bool = false
```

Comment:
[nits]
初期値を指定する場合は型推論が効くので、型指定を省略することが多いです。

```suggestion
    @AppStorage("hasCompletedOnboarding") var hasCompletedOnboarding = false
```

Reply:
こちらで修正しました
https://github.com/y-iwamoto/RemoteWellnessSupportApp/commit/b58902a5a00a82208cfb9d147e5a1c5500e2ec68